manifests/params.pp

```diff
@@ -59,7 +59,11 @@
   $config_file = "${config_dir}/cli.ini"
 
   if $facts['osfamily'] == 'RedHat' {
-    $configure_epel = true
+    if $facts['lsbdistid'] == 'Fedora' {
```

Comment:
Can be simpified to `$configure_epel = $facts['lsbdistid'] != 'Fedora'` but I'd prefer to use the built in `operatingsystem` fact. That doesn't rely on the lsb package being installed.

Reply:
I believe I have done as you have requested.